go/tools/builders/pack.go

```diff
@@ -330,3 +337,36 @@ func appendFiles(goenv *env, archive string, files []string) error {
 	args = append(args, files...)
 	return goenv.runCommand(args)
 }
+
+type fileMatcher func(name string) bool
+var fileNotFound = errors.New("file not found in archive")
```

Comment:
Rename to `errArchiveFileNotFound`. Named errors should usually start with `err` or `Err` like `os.ErrNotExist`.

Reply:
Can I reuse `os.ErrNotExist` instead of defining a new one?

Reply:
Sure, that seems fine.